kolibri/core/assets/src/views/content-icon/index.vue

```diff
@@ -97,6 +116,29 @@
       colorClass() {
         return `color-${this.colorStyle}`;
       },
+      tooltipText() {
+        const kind = this.kind;
+        if (kind === Constants.ContentNodeKinds.TOPIC) {
+          return this.$tr('topic');
+        } else if (kind === Constants.ContentNodeKinds.CHANNEL) {
+          return this.$tr('channel');
+        } else if (kind === Constants.ContentNodeKinds.EXERCISE) {
+          return this.$tr('exercise');
+        } else if (kind === Constants.ContentNodeKinds.VIDEO) {
+          return this.$tr('video');
+        } else if (kind === Constants.ContentNodeKinds.AUDIO) {
+          return this.$tr('audio');
+        } else if (kind === Constants.ContentNodeKinds.DOCUMENT) {
+          return this.$tr('document');
+        } else if (kind === Constants.ContentNodeKinds.HTML5) {
+          return this.$tr('html5');
+        } else if (kind === Constants.ContentNodeKinds.EXAM) {
+          return this.$tr('exam');
+        } else if (kind === Constants.ContentNodeKinds.LESSON) {
+          return this.$tr('lesson');
+        }
+        return this.$tr('item');
```

Comment:
Item would never be used since there is no icon we fallback to?

Reply:
would you prefer it return an empty string?

Reply:
To avoid adding an extra string to translate, yes.